code/src/layouts/header/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import AppBox from "@/components/app-box/AppBox";
+
+import HeaderToolbar from "@/layouts/header/components/header-toolbar/HeaderToolbar";
+import HeaderCategories from "@/layouts/header/components/header-categories/HeaderCategories";
+import useStickyHeader from "@/layouts/header/hooks/use-sticky-header/useStickyHeader";
+
+import "@/layouts/header/Header.scss";
+
+const Header = () => {
+  const { headerRef, scrollHandleRef } = useStickyHeader();
+
+  return (
+    <>
+      <AppBox ref={scrollHandleRef} data-scroll-handle />
```

Comment:
what is the purpose `data-scroll-handle` prop here ?

Reply:
I followed guide on youtube: https://www.youtube.com/watch?v=V-CBdlfCPic

I added this data attribute to visually highlight this box among other elements on the page in inspector tab